src/ipc/StoreMap.h

```diff
@@ -73,7 +73,9 @@ class StoreMapAnchor
 
 public:
     mutable ReadWriteLock lock; ///< protects slot data below
-    std::atomic<uint8_t> waitingToBeFreed; ///< may be accessed w/o a lock
```

Comment:
Please do not change this type unless we have to for some reason. The correct type here may actually be std::atomic_bool, but this PR is not about improving that type and such improvements are not trivial.

Reply:
Still open.

Reply:
Adding this old problem to the new review: Please do not change this type unless we have to for some reason. The correct type here may actually be std::atomic_bool, but this PR is not about improving that type and such improvements are not trivial.